src/hyperion/external_interaction/callbacks/rotation/ispyb_callback.py

```diff
@@ -38,8 +38,12 @@ class RotationISPyBCallback(BaseISPyBCallback):
     Usually used as part of a RotationCallbackCollection.
     """
 
-    def __init__(self) -> None:
-        super().__init__()
+    def __init__(
```

Comment:
Should: As with the centring callback I think we can call the base methods and let them do the tagging

Reply:
good point, except for the `stop` method which we can only use once, I've made it so that we only use the base class for tagging